package-lock.json

```diff
@@ -1832,6 +1832,7 @@
         "anymatch": "^2.0.0",
         "async-each": "^1.0.1",
         "braces": "^2.3.2",
+        "fsevents": "^1.2.7",
```

Comment:
Please check out a fresh copy of this file.

Reply:
Hi @apb7, This is a fresh copy I deleted package-lock.json and ran start.sh, but there were no changes made.
are the steps correct?

Reply:
I think this is because you're using MacOS. Can you please revert the changes in this file? Thanks!

Reply:
@apb7,  Yes sure.
Done PTAL Thanks!